k6ext/env.go

```diff
@@ -0,0 +1,40 @@
+package k6ext
+
+import (
+	"crypto/rand"
+	"math/big"
+	"os"
+	"strings"
+)
+
+// envResolve is defined as global var so we can overwrite it in
+// unit tests and therefore not manipulate the environment, which
+// allows us to run the test in parallel safely.
+var envResolve envResolver = os.LookupEnv //nolint:gochecknoglobals
+
+type envResolver func(key string) (string, bool)
```

Comment:
I though about this, but I saw pros and cons. With this implementation we would support parallelism in test cases and avoid the global var, but we would "complicate" more the "runtime" code.
If you think that's the better option I'm happy to change it! :+1:  Just pointing out my though process for this decision.

Reply:
I think this is perfectly fine to "complicate" the code a little instead having a global nasty environment variable. I don't worry too much about the current solution because we'll soon get rid of it.

Reply:
Got it. Nevermind, I think we can change this now also.
See [ec5399b](https://github.com/grafana/xk6-browser/pull/819/commits/ec5399bfd420498d6e731f9abd77a4fe60c8b3b9) and [953dee6](https://github.com/grafana/xk6-browser/pull/819/commits/953dee6ee05e6377217d5bfb41ea08441b49bf7b).

Reply:
Thanks for considering and applying the change 👍